lib/oban/migrations.ex

```diff
@@ -441,4 +441,89 @@ defmodule Oban.Migrations do
       record_version(prefix, 7)
     end
   end
+
+  defmodule V9 do
+    @moduledoc false
+
+    use Ecto.Migration
+
+    import Oban.Migrations.Helper
+
+    def up(prefix) do
+      alter table(:oban_jobs, prefix: prefix) do
+        add_if_not_exists(:meta, :map)
+        add_if_not_exists(:cancelled_at, :utc_datetime_usec)
+      end
+
+      execute """
+      DO $$
+      BEGIN
+      IF EXISTS (SELECT 1 FROM pg_type
```

Comment:
For this migration we can safely assume that the `oban_job_state` enum is available. There is no need to do the `IF EXISTS` check within the `DO $$` block.